### PR TITLE
slot machine effects

### LINKED ISSUE #107: Slot machine effects
**Is your feature request related to a problem? Please describe.**
Animation for the surrounding area in the slot machine when reel is pulled

**Describe the solution you'd like**
Keyframes and importing images from the assets folder

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Surrounding pokemon animation
2. Added slot machine blinkers

## Purpose
Give it more pizzazz

Closes #107

### CODE REVIEW

File: pokeapp/src/styling/components/_gamepage.scss [hunk 1]
```diff
@@ -164,3 +239,88 @@
...
+    animation-name: top-wave;
+    animation-duration: 1.5s;
+    animation-iteration-count: infinite;
+}
+
+.top-1:nth-child(odd) {animation-delay: 0.5s;}
+.top-1:nth-child(even) {animation-delay: 0.7s;}
+.top-2:nth-child(odd) {animation-delay: 0.9s}
+.top-2:nth-child(even) {animation-delay: 0.3s}
+
```
Comment (krferrell) [inline pokeapp/src/styling/components/_gamepage.scss:260]: Nice work with these top tier CSS skills.

**Review by krferrell — APPROVED** [review]
These animations are looking so sick. Solid work here.

File: pokeapp/src/components/GamePage.js [hunk 2]
```diff
@@ -136,56 +136,122 @@ const GamePage = ({
...
+                    <div className={pullLever(`hole`)}/>
+
+                    <div className={pullLever(`stick`)}>
+                      <div 
+                        className="lever-ball"
+                        onClick={() => {
+                          setActive(!reel1)
+                          setTimeout(() => setReel2(!reel2), 200);
+                          setTimeout(() => setReel3(!reel3), 400);
+                          setTimeout(() => winAPokemon(pokedex), 3400);
```
Comment (BlackCubes) [inline pokeapp/src/components/GamePage.js:240]: Nice usages of `setTimeout`. Curious to see this working.

**Review by BlackCubes — APPROVED** [review]
Looks pretty noice! Interested to see those `setTimeouts` in action 🥳

**Review by connielion — APPROVED** [review]
Looks great!

**Pull request merged**